src/libawkward/array/ListArray.cpp

```diff
@@ -1740,7 +1740,7 @@ namespace awkward {
     Index64 nextcarry(numvalid);
     Index64 smalloffsets(slicestarts.length() + 1);
     Index64 largeoffsets(slicestarts.length() + 1);
-    struct Error err2 = awkward_listarray_getitem_jagged_shrink_64(
+    struct Error err2 = kernel::listarray_getitem_jagged_shrink<int64_t>(
```

Comment:
This one was missed: it should be untemplated and the name should be `kernel::listarray_getitem_jagged_shrink_64`.